rpcserver.go

```diff
@@ -0,0 +1,60 @@
+package main
```

Comment:
Can you rename the current main package to btclightclient and your rpc logic to server package or something you suggestion? 
I think this will make more sense.

Reply:
I've updated the packages structure, please check if it works.